charts/vector/README.md

```diff
@@ -32,14 +32,20 @@ helm install --name <RELEASE_NAME> \
 
 ### Upgrading
 
+#### From vector-agent
+
+TODO
+
+#### From vector-aggregator
+
 TODO
 
 ### Uninstalling the Chart
 
 To uninstall/delete the `<RELEASE_NAME>` deployment:
 
 ```bash
-helm delete <RELEASE_NAME> --purge
+helm delete <RELEASE_NAME>
```

Comment:
Just curious why we dropped the `--purge`.

Reply:
`--purge` is only supported on Helm v2 and isn't an option on v3!